charts/gardener/controlplane/charts/application/templates/apiservice-v1alpha1-core-gardener-cloud.yaml

```diff
@@ -16,8 +16,8 @@ spec:
   {{- end }}
   group: core.gardener.cloud
   version: v1alpha1
-  groupPriorityMinimum: 10
-  versionPriority: 10
+  groupPriorityMinimum: 11
+  versionPriority: 11
```

Comment:
decrease these numbers below 9, or increase the default values here https://github.com/gardener/gardener/pull/1760/files#diff-74659eb77c2738c6f2ef2f80f3d87231R19-R20